index.yaml

```diff
@@ -1,9 +1,27 @@
 apiVersion: v1
 entries:
+  admission-controller:
+  - apiVersion: v2
+    appVersion: 0.0.2
+    created: "2021-10-14T17:30:09.643059-07:00"
+    description: Lacework admission controller using Lacework proxy scanner
+    digest: 4d5cd2204fcd14debf8069ef0b6824ec149f0fa28648b569f18b81fa83b1b69d
+    keywords:
+    - admission-controller
+    - runtime-security
+    - lacework
+    maintainers:
+    - email: chitra.kumar@lacework.net
+      name: chitrapkumar
+    name: admission-controller
+    type: application
+    urls:
+    - admission-controller-1.0.9.tgz
+    version: 1.0.9
   lacework-agent:
   - apiVersion: v1
     appVersion: "1.0"
-    created: "2021-10-06T20:28:07.439483296-07:00"
+    created: "2021-10-14T17:30:09.643832-07:00"
```

Comment:
Can you revert this change in the `index.yaml` file? In theory, using `--merge` should have left this alone and it looks like I need to update our instructions.